packages/core/src/elements/ResponsiveElement.ts

```diff
@@ -31,14 +31,23 @@ export type ResizeEvent = CustomEvent<{
  * @returns {void}
  */
 const triggerResize = (entry: ResizeObserverEntry): void => {
-
-  const { inlineSize: width, blockSize: height } = entry.borderBoxSize[0];
+  let width: number;
+  let height: number;
+  /* istanbul ignore else */
```

Comment:
let's update to the syntax of v7 code coverage tool.

Reply:
fixed.